multimodal/tests/unittests/others_2/test_predictor_with_bart.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+import warnings
+
+import numpy.testing as npt
+import pytest
+from datasets import load_dataset
+from torch import Tensor
+
+from autogluon.multimodal import MultiModalPredictor
+from autogluon.multimodal.constants import IA3_LORA
+
+
+@pytest.mark.parametrize("checkpoint_name", ["facebook/bart-base"])
+@pytest.mark.parametrize("efficient_finetune", [None, IA3_LORA])
+def test_predictor_with_bart(checkpoint_name, efficient_finetune):
+    train_data = load_dataset("glue", "mrpc")["train"].to_pandas().drop("idx", axis=1).sample(500)
+    test_data = load_dataset("glue", "mrpc")["validation"].to_pandas().drop("idx", axis=1).sample(20)
+    predictor = MultiModalPredictor(label="label")
+    predictor.fit(
+        train_data,
+        hyperparameters={
+            "model.hf_text.checkpoint_name": "yuchenlin/BART0",
```

Comment:
Should be `"model.hf_text.checkpoint_name": checkpoint_name `?

Reply:
@zhiqiangdon Just see this.. let me fix it in a follow-up PR about fixing the TIMM custom image size.